pkg/testutil/testutil.go

```diff
@@ -0,0 +1,67 @@
+// The MIT License (MIT)
```

Comment:
Why not just https://github.com/stretchr/testify/tree/master/assert + https://github.com/stretchr/testify/tree/master/require? (:

Reply:
He, fun story. So I actually rolled out those libs to a significant fraction of Prometheus code bases. We had a longer upstream discussion recently and concluded that the diffs it shows are rarely readable, which removes one expected benefit. Additionally, the rich equality checks were often behaving in unexpected ways. This caused us to work around it in non-obvious ways.

Overall, all we wanted was short versions for "no error occurred" and `reflect.DeepEqual`. Those few simple functions give us just that and are easy to use. Thus we decided to migrate Prometheus upstream to those.

Reply:
> Additionally, the rich equality checks were often behaving in unexpected ways. 
We did not notice these.. I guess only for some edge cases, right?

OK, so let's stick to these, I will try to limit usage to only these four ^^